qiskit/mapper/_mapping.py

```diff
@@ -263,51 +386,48 @@ def direction_mapper(circuit_graph, coupling_graph):
     return circuit_graph
 
 
-def update_qasm(i, first_layer, best_layout, best_d,
-                best_circ, circuit_graph, layer_list):
+def swap_mapper_layer_update(i, first_layer, best_layout, best_d,
+                             best_circ, layer_list,
+                             coupling):
```

Comment:
@awcross1 seems lilke this argument `coupling` is not being used in the function... wondering if this might be a bug, or we can just get rid of it.

Reply:
@atilag thanks for catching this. i don't think we need it.